remotes/fixup.go

```diff
@@ -110,23 +110,21 @@ type FixupOption func(*fixupConfig) error
 
 // ResolverConfig represents a resolver and its associated OriginProviderWrapper
 type ResolverConfig struct {
-	Resolver              remotes.Resolver
-	OriginProviderWrapper OriginProviderWrapper
+	Resolver remotes.Resolver
```

Comment:
Is that type still required since it only contains a resolver?

Reply:
Not really actually, I'll remove it